tools/logger/convert.c

```diff
@@ -640,10 +640,13 @@ static int logger_read(void)
 					return -errno;
 				}
 			}
```

Comment:
@ktrzcinx Fixes tag should look like this:

```
Fixes: 337afc4: ("logger: add error message when reading sof/etrace instead of sof/trace")
```

Reply:
Updated